test/v1beta1/resources/operator.knative.dev_v1beta1_knativeserving_cr.yaml

```diff
@@ -5,7 +5,10 @@ metadata:
 spec:
   config:
     network:
-      internal-encryption: "true"
+      # we currently cannot run with encryption enabled
+      # as the net-certmanager stuff has not yet been integrated into Serving
+      # TODO: https://issues.redhat.com/browse/SRVKS-1198
+      internal-encryption: "false"
```

Comment:
Good to have it there.

Reply:
yeah, fails without it, as long as we do not have net-certmanager yet :)